src/go/configgenerator/filtergen/backend_auth.go

```diff
@@ -53,6 +53,8 @@ type BackendAuthGenerator struct {
 
 	AccessToken *helpers.FilterAccessTokenConfiger
 
+	CorsUtility
```

Comment:
nit: no need for its own struct. How about capture variable on line 52 as `CORSOperationLimiter: string`